code/frameworks/react-vite/src/plugins/docgen-handlers/actualNameHandler.ts

```diff
@@ -11,32 +11,27 @@
 
 import { namedTypes as t } from 'ast-types';
 import type { NodePath } from 'ast-types/lib/node-path';
-import { getNameOrValue, isReactForwardRefCall } from 'react-docgen/dist/utils';
-import type { Importer } from 'react-docgen/dist/parse';
-import type Documentation from 'react-docgen/dist/Documentation';
+import { utils } from 'react-docgen';
+import type { Documentation } from 'react-docgen';
 
-export default function actualNameHandler(
-  documentation: Documentation,
-  path: NodePath,
-  importer: Importer
-): void {
+export default function actualNameHandler(documentation: Documentation, path: NodePath): void {
   if (t.ClassDeclaration.check(path.node) || t.FunctionDeclaration.check(path.node)) {
-    documentation.set('actualName', getNameOrValue(path.get('id')));
+    documentation.set('actualName', utils.getNameOrValue(path.get('id')));
   } else if (
     t.ArrowFunctionExpression.check(path.node) ||
```

Comment:
alpha.4 did move away from `ast-types` and instead uses babel-types

In this line as an example it would be 

```suggestion
    path.isArrowFunctionExpression() ||
```

I know alpha 4 is a huge change and I would actually advice to wait a little bit more with the update, I have 2 more breaking changes in mind that I want to get into v6.
After that I'm happy to help migrate to the new version. Feel free also to ping me if something is unclear in the new version.

Reply:
@danez that's awesome! we're planning a huge re-architecture of how we deal with "annotations" (what we call the generic form of docgen info) next year. We'll probably begin that work in late Q1 so it would be great to sync up beforehand. Would you be up for a short meeting in January?

Reply:
Yeah sure, feel free to reach out.

Reply:
I will! Thank you! 🙏